output/pickle.py

```diff
@@ -0,0 +1,131 @@
+import alchemlyb.preprocessing
+import pandas
+import numpy as np
+import pickle
+import time
+import os
+
+class Pickle:
+    name = 'pickle'
+    k_b = 8.3144621E-3
```

Comment:
If we need constants then we should have them all in _one_ place.

Reply:
It would be possible to use `scipy.constants` since scipy is a dependency of alchemlyb anyway?

Reply:
Yes, good idea.

Reply:
(You'll still have to add it to flamel's setup.py to be explicit.)

Reply:
Er, sorry... there's no setup.py. In fact there's no installation at all... see #3